surveys/templates/survey_submitted_detail.html

```diff
@@ -97,16 +104,14 @@ <h3 class="card-title">Analyze</h3>
 <script src="{% static 'js/jquery-ui-1.21.1.min.js' %}" type="text/javascript"></script>
 <script>
 $(document).ready(function () {
-  $('#survey-submitted-detail').DataTable({
-    // scrollx styling taken from
-    // https://stackoverflow.com/questions/45136218/datatables-scrollx-causing-squashed-header
-    "sScrollX": "100%",
-    "sScrollXInner": "110%",
-    "fixedColumns":   {
-      "leftColumns": 1
-    }
+  var table = $('#survey-submitted-detail').DataTable({
+    buttons: [{ extend: 'csv', text: 'Download as CSV' }],
```

Comment:
Because I'm a sucker for icons, how would you feel about a download icon here?

```suggestion
    buttons: [{ extend: 'csv', text: '<i class="fa fa-fw fa-download mr-1"></i>Download as CSV' }],
```

![Screenshot from 2019-06-03 16-47-49](https://user-images.githubusercontent.com/14170650/58836784-57ffc980-861f-11e9-9a1e-d871fed2ae74.png)

Reply:
It'd also be nice to use the `filename` attribute to set the CSV filename based on the title of the survey.